src/service/GraphQLService/index.ts

```diff
@@ -33,7 +33,7 @@ export class GraphQLService {
     this.keycloak = opts.keycloak;
   }
 
-  async sendQuery<T>(query: GraphQLQueryObject, fetchOpts?: RequestInit): Promise<GraphQLResponse<T>> {
+  async sendQuery<T>(query: GraphQLQueryObject, fetchOpts?: RequestInit): Promise<any> {
```

Comment:
Why `any`?

Reply:
Just forgot to change it back :) Fixed in 5671c02